tests/e2e/specs/in-game/gameOver.spec.js

```diff
@@ -383,6 +383,22 @@ describe('Stalemates', () => {
         .find('[data-cy=reject-stalemate]')
         .click();
     });
+
+    it('Asks for stalemate then disconnects before opponent reconnects', () => {
```

Comment:
This should say... 'Player requests stalemate, then reloads before opponent accepts'